File: frontend/src/component/Issue.js
import React from 'react';

const Issue = (props) => {
    const { issueTitle, issueDescription, issueId } = props.issue;

    
    return (
        <div>
            <a href={`/issue/${issueId}`}>
                <div>
                    <h5>{issueTitle}</h5>
                    <p>{issueDescription}</p>
                </div>
            </a>
        </div>
    );
};

export default Issue;

File: backend/src/main/kotlin/com/sitemate/models/Issue.kt
package com.sitemate.models

import kotlinx.serialization.Serializable

@Serializable
data class Issue(
    val issueId: Int,
    var issueTitle: String,
    var issueDescription: String
){
    init {
        require(issueTitle.isNotEmpty()){
            "Issue title cannot be empty"
        }
        require(issueDescription.isNotEmpty()){
            "Issue description cannot be empty"
        }
    }
}
